job-dsls/jobs/prod/prod_bamoe_properties_generator.groovy

```diff
@@ -0,0 +1,77 @@
+/**
+ * Generate properties files for productized builds.
+ *
+ * Generated files for productized builds:
+ * - {rhpam}-deliverable-list-staging.properties - properties file pointing binaries in the staging area
+ * - {rhpam}-deliverable-list.properties - properties file pointing binaries in the candidates area
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def propGen ="""
+import groovy.json.JsonOutput
+
+node('${Constants.LABEL_KIE_RHEL}') {
+
+    sh 'env\'
+    def binding = JsonOutput.toJson([
+            "REPO_URL"                   : REPO_URL,
+            "DELIVERABLE_REPO_URL"       : DELIVERABLE_REPO_URL,
+            "PRODUCT_VERSION"            : PRODUCT_VERSION,
+            "PRODUCT_VERSION_LONG"       : PRODUCT_VERSION_LONG,
+            "PRODUCT_MILESTONE"          : PRODUCT_MILESTONE,
+            "BOM_VERSION"                : BOM_VERSION,
+            "KIE_VERSION"                : KIE_VERSION,
+            "MVEL_VERSION"               : MVEL_VERSION,
+            "IZPACK_VERSION"             : IZPACK_VERSION,
+            "INSTALLER_COMMONS_VERSION"  : INSTALLER_COMMONS_VERSION,
+            "JAVAPARSER_VERSION"         : JAVAPARSER_VERSION
+    ])
+    def folder = "bamoe/bamoe-\${PRODUCT_VERSION}.\${PRODUCT_MILESTONE}"
+
+    build job: env.PROPERTIES_GENERATOR_PATH, parameters: [
+        [\$class: 'StringParameterValue', name: 'FILE_ID', value: "\${RELEASE_STAGING_FILE_ID}"],
+        [\$class: 'StringParameterValue', name: 'FILE_NAME', value: 'rhpam-deliverable-list-staging.properties'],
```

Comment:
```suggestion
        [\$class: 'StringParameterValue', name: 'FILE_NAME', value: 'bamoe-deliverable-list-staging.properties'],
```

Reply:
We have been using the properties file with rhpam prefix not bamoe, that's why I kept like this

Reply:
I agree with @rgdoliveira 's proposal